ubuntu-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:12.04
+FROM node:argon
```

Comment:
I didn't confirme it but in shippify you use node:latest
Let me confirm that ok